tests/test_hvac_csc.py

```diff
@@ -151,7 +113,7 @@ async def _verify_telemetry(self, subsystem):
             # This topic only publishes a temperature so we skip it here.
             if name == "generalP01":
                 continue
-            if topic.value in xml.TOPICS_ALWAYS_ENABLED or name == subsystem:
+            if topic.value in TOPICS_ALWAYS_ENABLED or name == subsystem:
```

Comment:
Looking at this now I am surprised at "the *one* subsystem" (as per the comment on the next line). Naively naively this `if` looks like it would collect more than one topic because of the `or`. If you are really only expecting one match then do you need the `or name == subsystem` at all?

This shows up in several places.

Reply:
There are three devices that are always enabled and they must not be forgotten.